Source/WebCore/PAL/pal/cf/CoreMediaSoftLink.h

```diff
@@ -409,6 +411,19 @@ SOFT_LINK_CONSTANT_MAY_FAIL_FOR_HEADER(PAL, CoreMedia, kCMFormatDescriptionExten
 SOFT_LINK_CONSTANT_MAY_FAIL_FOR_HEADER(PAL, CoreMedia, kCMFormatDescriptionProjectionKind_Rectilinear, CFStringRef)
 #define kCMFormatDescriptionProjectionKind_Rectilinear get_CoreMedia_kCMFormatDescriptionProjectionKind_Rectilinear()
 
+namespace PAL {
+
+inline std::span<uint8_t> CMBlockBufferGetDataSpan(CMBlockBufferRef theBuffer, size_t offset = 0)
+{
+    char* data = nullptr;
+    size_t lengthAtOffset = 0;
+    if (auto error = PAL::CMBlockBufferGetDataPointer(theBuffer, offset, &lengthAtOffset, nullptr, &data))
+        return { };
+    return byteCast<uint8_t>(unsafeMakeSpan(data, lengthAtOffset));
```

Comment:
I'd be tempted to do the `byteCast` on the pointer rather than the `span` to reduce the chance of unnecessary copying.